framework/lib/components/toolbox/toolbox.tsx

```diff
@@ -40,19 +40,22 @@ import { getChildrenWithProps } from '../../utils'
  }
  * ?)
  */
+
+const clamp = (minValue: number, maxValue: number, value: number) =>
```

Comment:
The PR is good to go, but could you please move this function before the documentation comment. Because of the placement of this function, the documentation does not render the example component :)